pytest_circleci_parallelized.py

```diff
@@ -16,13 +16,21 @@ def pytest_addoption(parser):
     )
 
 
+def get_verbosity(config):
+    return config.option.verbose
```

Comment:
What happens here if the user hasn't manually specified a "verbose" option in their config? Will it be set automatically by CircleCI?

Reply:
pytest default is 0

Reply:
That is tested in the current test cases.